department-of-veterans-affairs/caseflow
# JSON Exporter: include organization admins in export

[Slack](https://dsva.slack.com/archives/C2ZAMLK88/p1626383564285000?thread_ts=1626380458.279800&cid=C2ZAMLK88)

### Description
When exporting appeals via [JSON Exporter](https://github.com/department-of-veterans-affairs/caseflow/wiki/Exporting-and-Importing-Appeals), include organization `admins` in the export so that the Team Management page works.

### Acceptance Criteria
- [x] the Team Management page works on an exported appeal

### Testing Plan
- Export Appeal id=53008 from prod
- Import it as seed data or in an RSpec and check the Team Management page works
